setup/www/resources/config/nodejs.org

```diff
@@ -354,8 +354,10 @@ server {
     rewrite ^/feed/vulnerability/?$                               https://$server_name/en/feed/vulnerability.xml permanent;
 
     # Asset rewrites
+    rewrite ^/layouts/css/styles\.css$                            https://$server_name/static/css/styles.css permanent;
     rewrite ^/(?!static/)?favicon\.ico$                           https://$server_name/static/images/favicons/favicon.ico permanent;
-    rewrite ^/apple-touch-icon.*\.png$                            https://$server_name/static/images/favicons/apple-touch-icon.png permanent;
+    rewrite ^/(?!static/)?favicon\.png$                           https://$server_name/static/images/favicons/favicon-32x32.png permanent;
+    rewrite ^/(?!static/)?apple-touch-icon.*\.png$                https://$server_name/static/images/favicons/apple-touch-icon.png permanent;
```

Comment:
why did this one need to change? was there apple-touch-icon*.png files in other directories other than / and /static/?

Reply:
There are requests to `https://nodejs.org/static/apple-touch-icon.png` which isn't handled without this change. I noticed this on https://github.com/nodejs/build/pull/1936/files#diff-5322f429725c524a1cbc315351962ec4L24

Reply:
Actually, wait I think my regex is wrong

Reply:
We basically need the regex to work for paths with and without `static`